### DISCUSSION TITLE
Setting Security Group When Creating Environment

### DISCUSSION THREAD

**Discussion by just-marc-stevenson** [microsoft/powerplatform-build-tools#535] [Ideas]
When using `pac admin create` there is not an option to set the Security Group to secure access to that environment.

Microsoft.PowerApps.Administration.PowerShell New-AdminPowerAppEnvironment command has this option and this omission is the only thing stopping us moving over to using the pac cli instead. Is this option supported and something that I've missed/upcoming/no longer supported?

Thanks

**Comment by allandecastro** [comment]
+1 on this

**Comment by mrmwink** [comment]
Agreed, we have a similar challenge

**Comment by petrochuk** [comment]
Converting to discussion so people can vote on it

**Comment by mVermaat** [comment]
Same is needed for the copy/restore. Good change.

**Comment by prohannes** [comment]
What about `pac admin reset`?

**Comment by allandecastro** [comment]
Hello
CLI can now handle this adding this to the admin create:
![image](https://github.com/microsoft/powerplatform-build-tools/assets/9749002/c8940824-345c-4e9d-8c67-a47198c4abab)
